source/getting-started/autostart-systemd.markdown

```diff
@@ -35,6 +35,23 @@ WantedBy=multi-user.target
 EOF'
 ```
 
+If you've setup `hass` in virtualenv following the guide the following template should work for you.
```

Comment:
I would replace `hass` in this context with Home Assistant to prevent confusion.